test/dom/Value.ts

```diff
@@ -1,89 +1,93 @@
-    dom.Struct
+  dom.Null,
+  dom.Boolean,
+  dom.Integer,
+  dom.Float,
+  dom.Decimal,
+  dom.Timestamp,
+  dom.Clob,
+  dom.Blob,
+  dom.String,
+  dom.Symbol,
+  dom.List,
+  dom.SExpression,
+  dom.Struct,
 ];
 
 // Converts each argument in `valuesToRoundTrip` to a dom.Value, writes it to an Ion stream, and then load()s the Ion
 // stream back into a dom.Value. Finally, compares the original dom.Value with the dom.Value load()ed from the stream
 // to ensure that no data was lost.
-function domRoundTripTest(typeName: string,
-                          equalityTest: (p1, p2) => boolean,
-                          ...valuesToRoundTrip: any[]) {
-    describe(typeName, () => {
-        for (let value of valuesToRoundTrip) {
-            it(value + '', () => {
-                let writer = ion.makeBinaryWriter();
-                // Make a dom.Value out of the provided value if it isn't one already
-                let domValue;
-                if (value instanceof Value) {
-                    domValue = value as Value;
-                } else {
-                    domValue = Value.from(value);
-                }
-                // Serialize the dom.Value using the newly instantiated writer
-                domValue.writeTo(writer);
-                writer.close();
-                // Load the serialized version of the value
-                let roundTripped = load(writer.getBytes())!;
-                // Verify that nothing was lost in the round trip
-                assert.isNotNull(roundTripped);
-                assert.equal(domValue.isNull(), roundTripped.isNull());
-                assert.equal(domValue.getType(), roundTripped.getType());
-                assert.deepEqual(domValue.getAnnotations(), roundTripped.getAnnotations());
-                assert.isTrue(equalityTest(domValue, roundTripped));
-            });
+function domRoundTripTest(typeName: string, ...valuesToRoundTrip: any[]) {
+  describe(typeName, () => {
+    for (let value of valuesToRoundTrip) {
+      it(value + "", () => {
+        let writer = ion.makeBinaryWriter();
+        // Make a dom.Value out of the provided value if it isn't one already
+        let domValue;
+        if (value instanceof Value) {
+          domValue = value as Value;
+        } else {
+          domValue = Value.from(value);
         }
-    });
+        // Serialize the dom.Value using the newly instantiated writer
+        domValue.writeTo(writer);
+        writer.close();
+        // Load the serialized version of the value
+        let roundTripped = load(writer.getBytes())!;
+        // Verify that nothing was lost in the round trip
+        assert.isNotNull(roundTripped);
+        assert.equal(domValue.isNull(), roundTripped.isNull());
+        assert.equal(domValue.getType(), roundTripped.getType());
+        assert.deepEqual(
+          domValue.getAnnotations(),
+          roundTripped.getAnnotations()
+        );
+        assert.isTrue(domValue.equals(roundTripped));
```

Comment:
Presumably this last line takes care of all the asserts above it and they can be removed?

Reply:
yes. I will remove these unnecessary asserts.